tests.js

```diff
@@ -138,6 +150,21 @@ module.exports = testCase({
       test.done()
     },
 
+    'new Bandit with array should create the arms initialized properly': function(test) {
+
+      var bandit = new Bandit([{count: 100, sum: 20},{count: 50, sum: 30},{count:70, sum: 10}])
+
+      test.equal(bandit.arms.length, 3)
+      test.equal(bandit.arms[0].count, 100);
+      test.equal(bandit.arms[0].sum, 20);
+      test.equal(bandit.arms[1].count, 50);
+      test.equal(bandit.arms[1].sum, 30);
+      test.equal(bandit.arms[2].count, 70);
+      test.equal(bandit.arms[2].sum, 10);
```

Comment:
Sorry I realize this style is a little strange to some eyes but would you mind removing the semicolons (here and elsewhere) to make it consistent with the rest of the code?